content/_index.md

```diff
@@ -36,6 +36,17 @@ grid:
         url: "/build/building/packaging/"
       - text: "Release notes"
         url: "/build/release-notes/"
+  - title: Docker Build Cloud
+    icon: cloud
+    description: |
+      Run your builds in the cloud.
+    links:
+      - text: "Overview"
+        url: "/build-cloud/"
```

Comment:
could we take the opportunity to add new labels for this content and remove `area/build` from it? :sweat_smile:

Reply:
good catch!